docs/docsite/rst/playbook_guide/playbooks_delegation.rst

```diff
@@ -12,6 +12,8 @@ Tasks that cannot be delegated
 ------------------------------
 
 Some tasks always execute on the control node. These tasks, including ``include``, ``add_host``, and ``debug``, cannot be delegated.
+You can determine if an action can be delegated from the ``connection`` attribute documentation.
+If the ``connection`` attribute indicates ``support`` is ``False`` or ``None``, then the action does not use a connection and cannot be delegated.
```

Comment:
I know this is already merged, but just a note that there was a big push from the docs folks to change every boolean value to lowercase `true` and `false` (I guess except where it's literal Python code?). Maybe I'm misremembering, but might be worth checking with them.

Reply:
Hey @briantist - just to circle back, we did have [a PR that ](https://github.com/ansible/ansible-documentation/pull/543) tried to fix this, but ends up it's not the same sort of thing as the boolean true/false...so we closed that pr w/o merging.